source/app.d

```diff
@@ -46,17 +49,16 @@ class ImageFitness
         destination = loadImage(path);
     }
 
-    void rasterize(ref SuperImage image, in Circle[] circles)
+    void rasterize(ref SuperImage image, in Rect[] rects)
     {
-        foreach(ref circle; circles)
+        foreach(ref rect; rects)
         {
-            auto color = circle.color;
-            foreach(x; (circle.x - circle.radius / 2) .. (circle.x + circle.radius / 2))
+            auto color = rect.color;
+            foreach(x; (rect.x) .. (rect.x + rect.width))
```

Comment:
Hele dčko má style guide... To je mi novinka, asi bych měl zase udělat nějaký refactoring...